<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Article</class>
 <widget class="QWidget" name="Article">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Article</string>
  </property>
  <widget class="QToolButton" name="OK">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QToolButton" name="NO">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="Name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>论文名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameIn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="Save">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <action name="actionOK">
   <property name="text">
    <string>OK</string>
   </property>
  </action>
  <action name="actionNO">
   <property name="text">
    <string>NO</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
